### ISSUE TITLE
Fix event list ajax issues

### ISSUE THREAD

**Issue by mikehand** [tbpmig/mig-website#185]
So far noticed:
- people "signed up" when not actually
- if event is not members only says need to log in
- asks to add some events to gcal even though should be done automatically

Data appears to be all correct.

**Comment by patosai** [comment]
Any ETA on when this will be fixed?

**Comment by mikehand** [comment]
No. Problem is complicated, low incidence, and difficult to reproduce. If
you have information that is useful in ascertaining the root cause, please
make it known.

ETAs are not a thing for website bugs, as a rule.

On Wednesday, October 7, 2015, Patrick Tsai <notifications@github.com>
wrote:

> Any ETA on when this will be fixed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tbpmig/mig-website/issues/185#issuecomment-146300668>.
>


-- 
Michael Hand
Electrical Engineering/Control Systems M.S.E.
The University of Michigan
mikehand@umich.edu

**Comment by patosai** [comment]
The ETA was to figure out if anyone was working on it, since the issue was created many moons ago. I'm noticing a lot of the "need to log in" bug.

Also on the Schlumberger info session, the div to edit/delete shows (though the actions don't go through).

**Comment by mikehand** [comment]
You don't have those permissions. Screenshots or the info is useless

On Wednesday, October 7, 2015, Patrick Tsai <notifications@github.com>
wrote:

> The ETA was to figure out if anyone was working on it, since the issue was
> created many moons ago. I'm noticing a lot of the "need to log in" bug.
>
> Also on the Schlumberger info session, the div to edit/delete shows
> (though the actions don't go through).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tbpmig/mig-website/issues/185#issuecomment-146303842>.
>


-- 
Michael Hand
Electrical Engineering/Control Systems M.S.E.
The University of Michigan
mikehand@umich.edu

**Comment by patosai** [comment]
![screen shot 2015-10-07 at 3 39 05 pm](https://cloud.githubusercontent.com/assets/5304405/10348874/9a744c06-6d09-11e5-8d8c-fc7df814744b.png)
![screen shot 2015-10-07 at 3 39 12 pm](https://cloud.githubusercontent.com/assets/5304405/10348875/9a79172c-6d09-11e5-89c7-f8898298d644.png)

These are just some of the events that have those options showing up.

**Comment by mikehand** [comment]
I'll also need your current position/role.
Also browser. Also if you've done anything abnormal like turn JavaScript off

On Wednesday, October 7, 2015, Patrick Tsai <notifications@github.com>
wrote:

> [image: screen shot 2015-10-07 at 3 39 05 pm]
> <https://cloud.githubusercontent.com/assets/5304405/10348874/9a744c06-6d09-11e5-8d8c-fc7df814744b.png>
> [image: screen shot 2015-10-07 at 3 39 12 pm]
> <https://cloud.githubusercontent.com/assets/5304405/10348875/9a79172c-6d09-11e5-89c7-f8898298d644.png>
>
> These are just some of the events that have those options showing up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tbpmig/mig-website/issues/185#issuecomment-146305854>.
>


-- 
Michael Hand
Electrical Engineering/Control Systems M.S.E.
The University of Michigan
mikehand@umich.edu

**Comment by patosai** [comment]
Electee, Firefox 38.0.5, everything running as normal (no AdBlock, no NoScript, nothing).

**Comment by mikehand** [comment]
Ducking Firefox. Making I.E. Look sane

On Wednesday, October 7, 2015, Patrick Tsai <notifications@github.com>
wrote:

> Electee, Firefox 38.0.5, everything running as normal (no AdBlock, no
> NoScript, nothing).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tbpmig/mig-website/issues/185#issuecomment-146313312>.
>


-- 
Mike Hand

**Comment by mikehand** [comment]
Patrick, can you test in another browser and see if the problem repeats?

On Wednesday, October 7, 2015, Mike Hand <mikehand89@gmail.com> wrote:

> Ducking Firefox. Making I.E. Look sane
>
> On Wednesday, October 7, 2015, Patrick Tsai <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> Electee, Firefox 38.0.5, everything running as normal (no AdBlock, no
>> NoScript, nothing).
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/tbpmig/mig-website/issues/185#issuecomment-146313312>
>> .
>>
>
>
> --
> Mike Hand
>


-- 
Mike Hand

**Comment by patosai** [comment]
Opera and Safari both have the same problem.

**Comment by mikehand** [comment]
@dadrian Did the above referenced commit do anything to fix the issue? It looks like it's a misnumbering

**Comment by dadrian** [comment]
I'm confused as to how this commit numbering happened. I'll take a look after quals (e.g. after Tuesday)
